src/components/inputFields/TextInputOutlined.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import {
+    responsiveHeight,
+    responsiveWidth,
+    responsiveFontSize
+} from "react-native-responsive-dimensions";
+import { HelperText, TextInput } from 'react-native-paper';
+import { TextInputOutlinedInterface } from '../../interface/inputFields/TextInputOutlinedInterface';
+import { KeyboardTypeOptions } from 'react-native';
+
+export const TextInputOutlined: React.FC<TextInputOutlinedInterface & { onChangeText: (text: string) => void }> = ({
+    placeHolder = "Input Text Box",
+    backgroundColor = "#3D3E41",
+    mode = "outlined",
```

Comment:
Don't give Hardcoded value for colors

Reply:
Then what will be the default value? we have to set default value at initial. as per color theme we can chnage this at render time. what default value should i use?